themes/edx.org/lms/static/sass/partials/lms/theme/_dashboard.scss

```diff
@@ -1,56 +1,196 @@
+.modal-open {
```

Comment:
Is everything left here styling we could not convert to utility classes? Or is the CSS cleanup still something we need to do in a future release?

Reply:
CSS cleanup is going to have to happen at some point. There are still things I have not yet transferred over.